docs/en/api/S3-API.md

```diff
@@ -23,61 +23,113 @@ Alluxio proxy, introducing an extra hop. For optimal performance, it is recommen
+      <ul style="list-style-type:none;margin:0;padding:0">
+      {% for param in params %}
+      {% if forloop.last %}
+      <li>{{ param }}</li>
+      {% else %}
+      <li>{{ param }},</li>
+      {% endif %}
+      {% endfor %}
+      </ul>
+      {% endif %}
+    </td>
   </tr>
 {% endfor %}
 </table>
 
-### Limitation
-In Alluxio, we use `/` as a reserved separator. Therefore, any S3 directory with an object named `/` (eg: `s3://example-bucket//`) will conflict and behave incorrectly.
-
-### Bucket
-Bucket must be a directory directly under a mount point.   
-If it is under a non-root mount point, the bucket separator must be used as the separator in the bucket name.   
-For example, mount:point:bucket represents Alluxio directory /mount/point/bucket.  
-
-## Language support
-Alluxio S3 client supports various programming languages, such as C++, Java, Python, Golang, and Ruby.
-In this documentation, we use curl REST calls and python S3 client as usage examples.
+### Limitations
+- S3 API clients must construct their HTTP requests by specifying the bucket name as the first path of the URI
+  (i.e: `http://s3.amazonaws.com/{bucket}/{object}`) and not as a subdomain of the host server
+  (i.e: `http://{bucket}.s3.amazonaws.com/{object}`).
+- **Only top-level Alluxio directories are treated as buckets by the S3 API**.
+  - Hence the root directory of the Alluxio filesystem is not treated as an S3 bucket.
+    Any root-level objects (eg: `alluxio://file`) will be inaccessible through the Alluxio S3 API.
+  - To treat sub-directories as a bucket, the separator `:` must be used in the bucket name (eg: `s3://sub:directory:bucket/file`).
+    - **Note that this is purely a convenience feature and hence is not returned by API Actions such as ListBuckets.**
+- Alluxio uses `/` as a reserved separator. Therefore, any S3 object named `/` (eg: `s3://example-bucket//`) will be inaccessible.
+- By default, the user that is used to perform any operations is the user that was used to
+  launch the proxy process. A different user can be specified through the `Credential` field of the
+  [AWS Authorization Header](https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html)
 
 ## Example Usage
 
 ### REST API
+
 For example, you can run the following RESTful API calls to an Alluxio cluster running on localhost.
 The Alluxio proxy is listening at port 39999 by default.
 
 #### Authorization
 
-By default, the user that is used to do any FileSystem operations is the user that was used to
-launch the proxy process. This can be changed by providing the Authorization Header. The header
-format is [defined by the AWS S3 Rest API reference](https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html)
+At the moment, access key and secret key validation does not exist for the Alluxio S3 API.
+Therefore the 'Authorization' header is used purely to specify the intended user to perform a request.
```

Comment:
single quote intentional or should be backtick?

Reply:
Outdated